src/build.py

```diff
@@ -1218,18 +1206,41 @@ def TestEmwasm():
       outdir=emscripten_wasm_out)
 
 
+def ExecuteEmscriptenTestSuite(name, config, outdir, warn_only):
+  buildbot.Step('Execute emscripten testsuite (%s)' % name)
+  Mkdir(outdir)
+  try:
+    proc.check_call(
+        [sys.executable,
+         os.path.join(INSTALL_DIR, 'emscripten', 'tests', 'runner.py'),
+         'binaryen2', '--em-config', config],
+        cwd=outdir)
+  except proc.CalledProcessError:
+    buildbot.Fail(warn_only)
+
+
 def TestEmtest():
   ExecuteEmscriptenTestSuite(
       'emtest',
```

Comment:
This should probably be "emwasm" to match the way the other step names are formatted.